app/src/main/assets/fido2_privileged_allow_list.json

```diff
@@ -519,6 +519,18 @@
           }
         ]
       }
+    },
+    {
+      "type": "android",
+      "info": {
+        "package_name": "org.cromite.cromite",
+        "signatures": [
+          {
+            "build": "release",
```

Comment:
Not sure if the build is release here? Same with CalyxOS Chromium, I'm assuming on both, not sure how to check this. @uazo

Reply:
I confirm the key I read.
I do not currently have a published test build, so yes, my published builds are releases.

I also plan to go out via google play, although I don't know when yet.
this is the key, if you prefer to enter it now:

`8F:D9:EF:18:1A:FD:06:9F:10:62:F0:82:23:BB:37:1A:58:6A:6D:33:EF:C2:5E:2C:40:FC:B7:1E:10:00:86:A0`